content/docs/start/data-and-model-versioning.md

```diff
@@ -174,9 +174,9 @@ $ rm -f data/data.xml
 </tab>
 <tab title="Windows">
 
-```dvc
-$ rmdir .dvc\cache
-$ del data\data.xml
+```powershell
+> rmdir .dvc\cache
+> del data\data.xml
```

Comment:
One problem is that `>` is rendered as part of the line so it can be copied to clipboard accidentally (doesn't happen to `$` under `dvc` highlighter).

Reply:
Prismjs doesn't do this by default on their `bash` or `powershell` code blocks. This feature is only on our custom dvc highlighter. If we wanted this feature on windows command blocks, we'd probably would have to create some kind of custom highlighter that builds off of powershell like you mentioned in your last comment in #3215 🤔

Reply:
Yeah we'd need it anyway for `dvc` commands. Unless we decide to just keep using the `dvc` highlighter and `$` prompt symbol as suggested by @shcheklein in https://github.com/iterative/dvc.org/pull/3215#discussion_r797874577 .

Reply:
That's a good idea! Then we wouldn't need to create a whole new highlighter! Though does `dvc` (which extends on `bash`) have any issues with highlighting windows related commands?

Reply:
I think there may be some false positives in highlights. But from the scope of commands we use I don't think that will pose much of a problem. The same highlighting can be used for both bash and PS.

Reply:
OK we're using `dvc` for Windows blocks too for now then. We can circle back on this later if needed.